DigitalLearningSolutions.Web/Views/Supervisor/ReviewCompetencySelfAsessment.cshtml

```diff
@@ -11,130 +11,156 @@
     </dd>
+    </div>
+  }
+  <div class="nhsuk-summary-list__row">
+  <dt class="nhsuk-summary-list__key">
+    Question
+  </dt>
+  <dd class="nhsuk-summary-list__value">
+    @Model.Competency.AssessmentQuestions.First().Question
+  </dd>
   </div>
 
   <div class="nhsuk-summary-list__row">
-    <dt class="nhsuk-summary-list__key">
-      Learner response
-    </dt>
-    <dd class="nhsuk-summary-list__value">
-      <partial name="Shared/_AssessmentQuestionResponse" model="Model.Competency.AssessmentQuestions.First()" />
-    </dd>
+  <dt class="nhsuk-summary-list__key">
+    Learner response
+  </dt>
+  <dd class="nhsuk-summary-list__value">
+    <partial name="Shared/_AssessmentQuestionResponse" model="Model.Competency.AssessmentQuestions.First()" />
+  </dd>
   </div>
   @if (Model.Competency.AssessmentQuestions.First().ResultRAG > 0)
   {
     <div class="nhsuk-summary-list__row">
-      <dt class="nhsuk-summary-list__key">
-        Role expectations
-      </dt>
-      <dd class="nhsuk-summary-list__value">
-        @(Model.Competency.AssessmentQuestions.First().ResultRAG == 1 ? "Not meeting" : Model.Competency.AssessmentQuestions.First().ResultRAG == 2 ? "Partially meeting" : "Fully meeting")
-      </dd>
+    <dt class="nhsuk-summary-list__key">
+      Role expectations
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @(Model.Competency.AssessmentQuestions.First().ResultRAG == 1 ? "Not meeting" : Model.Competency.AssessmentQuestions.First().ResultRAG == 2 ? "Partially meeting" : "Fully meeting")
+    </dd>
     </div>
   }
   @if (Model.Competency.AssessmentQuestions.First().IncludeComments)
   {
     <div class="nhsuk-summary-list__row">
-      <dt class="nhsuk-summary-list__key">
-        Comments
-      </dt>
-      <dd class="nhsuk-summary-list__value">
-        @Html.Raw(Model.Competency.AssessmentQuestions.First().SupportingComments)
-      </dd>
+    <dt class="nhsuk-summary-list__key">
+      Comments
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Html.Raw(Model.Competency.AssessmentQuestions.First().SupportingComments)
+    </dd>
     </div>
   }
   <div class="nhsuk-summary-list__row">
+  <dt class="nhsuk-summary-list__key">
+    Status
+  </dt>
+  <dd class="nhsuk-summary-list__value">
+    <partial name="Shared/_AssessmentQuestionStatusTag" model="Model.Competency.AssessmentQuestions.First()" />
+  </dd>
+  </div>
+  @if (Model.Competency.AssessmentQuestions.First().SelfAssessmentResultSupervisorVerificationId != null && Model.Competency.AssessmentQuestions.First().Verified == null && !(bool)Model.Competency.AssessmentQuestions.First().UserIsVerifier)
+  {
+    <div class="nhsuk-summary-list__row">
     <dt class="nhsuk-summary-list__key">
-      Status
+      Supervisor
     </dt>
     <dd class="nhsuk-summary-list__value">
-      <partial name="Shared/_AssessmentQuestionStatusTag" model="Model.Competency.AssessmentQuestions.First()" />
+      @Model.SupervisorName

```

Comment:
This appears to be showing the name of the logged in supervisor, not the supervisor to whom the request was submitted.

We need to add `SupervisorName` to the AssessmentQuestions model, altering the query that we use to populate AssessmentQuestions so that it loads the name of the supervisor that the request was submitted to. 

We can then display the name here using `@Model.Competency.AssessmentQuestions.First().SupervisorName`